app/util/cache.tsx

```diff
@@ -10,6 +11,8 @@ type Compressor = build.bazel.remote.execution.v2.Compressor.Value;
 const DigestFunction = build.bazel.remote.execution.v2.DigestFunction.Value;
 type DigestFunction = build.bazel.remote.execution.v2.DigestFunction.Value;
 
+const BYTESTREAM_URL_PATTERN = /^bytestream:\/\/(?<address>[^\/]+)\/((?<instanceName>.+?)\/)?(blobs|compressed-blobs\/(?<compressor>zstd))\/((?<ac>ac)\/)?((?<digestFunction>blake3)\/)?(?<hash>[0-9a-f]+)\/(?<sizeBytes>\d+)$/;
```

Comment:
The digest function match here kinda freaks me out because it only supports the one value that we currently support.

Maybe we need to start sharing test cases between client and server or something?  Then adding support for some new field to the server wouldn't break the client?